### PR TITLE
Anterior and posterior views

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
Supersedes Pr #1421 and #1559 to fix the docstring.

### CODE REVIEW

**Comment by GaelVaroquaux** [comment]
I am doing a PR to have circle run on the code.

**Comment by bthirion** [comment]
LGTM.

File: doc/whats_new.rst [hunk 1]
```diff
@@ -261,6 +261,8 @@ Enhancements
     - Extensive plotting example shows how to use contours and filled contours
       on glass brain.
 
+    - New 'anterior' and 'posterior' view added to the plot_surf family views
```
Comment (KamalakerDadi) [inline doc/whats_new.rst]: This is added in 0.3.beta

Comment (GaelVaroquaux) [reply doc/whats_new.rst]: Thanks. Fixed

**Comment by GaelVaroquaux** [comment]
Merging.

**Pull request merged**